apps/meteor/client/providers/SearchFiltersProvider/SearchFilterProvider.tsx

```diff
@@ -0,0 +1,16 @@
+import type { FC, ReactNode } from 'react';
+import React, { useState } from 'react';
+
+import type { SearchFilters } from '../../contexts/SearchFilterContext';
+import { SearchFilterContext } from '../../contexts/SearchFilterContext';
+
+type SearchFilterProviderProps = {
+	children: ReactNode;
```

Comment:
AFAIK: FC already have children described on it 🤔... there is no necessity of it here...